catalog/pom.xml

```diff
@@ -4416,6 +4416,7 @@
                             </extensionDirectories>
                             <skipArtifactIdBases>
                                 <skipArtifactIdBase>http-common</skipArtifactIdBase>
+                                <skipArtifactIdBase>qute</skipArtifactIdBase>
```

Comment:
Wouldn't this actually exclude the Qute component from the Camel Quarkus Catalog? - Is that what you want?

Reply:
I actually want to prevent the catalog from including a bogus `qute.json` that is causing this issue apache/camel-k-runtime#677. I'm adjusting the PR w/ your notes + @jamesnetherton' ones.

Reply:
Reading https://github.com/apache/camel-k-runtime/issues/677 I am still not 100% sure what is the problem. `org.apache.camel.component.qute.QuteComponent` exists in `org.apache.camel.quarkus:camel-quarkus-qute-component:jar` Don't you need to add it to your class path?

Reply:
I think the one bundled in the catalog jar is taking precedence in the classpath. Thus causing the issue on 667.

Reply:
Actually, we very recently started adding camel artifact coordinates to our catalog https://github.com/apache/camel-quarkus/issues/2368 I wonder whether that new `metadata: {camelArtifact: }` shouldn't be `org.apache.camel.quarkus:camel-quarkus-qute-component` for qute. Now it is probably empty, because it is a special case that is not handled properly.

Reply:
@ppalaga I'm leaning towards no, in this case, but I wouldn't know for sure. I think that, in the case of the Camel K catalog, and specifically in this problem, it is reading the `javaType` info and not the metadata.